src/features/import/hooks/useColumn.ts

```diff
@@ -56,37 +56,46 @@ export default function useColumn(orgId: number) {
     return !!exists;
   };
 
-  const columnOptions: Option[] = [];
+  const columnOptionsIDPartition: Option[] = [];
+  const columnOptionsFieldsPartition: Option[] = [];
+  const columnOptionsOtherPartition: Option[] = [];
 
-  columnOptions.push({
+  columnOptionsIDPartition.push({
     label: messages.configuration.mapping.id(),
     value: 'id',
   });
 
   Object.values(NATIVE_PERSON_FIELDS).forEach((fieldSlug) => {
     if (fieldSlug != 'id' && fieldSlug != 'ext_id') {
-      columnOptions.push({
+      columnOptionsFieldsPartition.push({
         label: globalMessages.personFields[fieldSlug](),
         value: `field:${fieldSlug}`,
       });
     }
   });
 
   customFields.forEach((field) =>
-    columnOptions.push({
+    columnOptionsFieldsPartition.push({
       label: field.title,
       value: `field:${field.slug}`,
     })
   );
 
-  columnOptions.push({
+  columnOptionsOtherPartition.push({
```

Comment:
Move this hard-coded array into `MappingRow` (see previous comment for reasoning).